services/postgres/serverless.yml

```diff
@@ -56,6 +56,11 @@ custom:
       - val
       - prod
       - main
+  pauseAurora:
+    other: true
+    dev: true
```

Comment:
I think this should be "main" we don't have a "dev" stage

Reply:
Ah yeah, of course. I actually just deleted `dev` altogether and let it fall back to the catchall `other`.